recipes/folly/all/conanfile.py

```diff
@@ -144,6 +151,9 @@ def source(self):
     def _configure_cmake(self):
         cmake = CMake(self)
         if can_run(self):
+            if self.options.use_sse4_2 and str(self.settings.arch) in ['x86', 'x86_64']:
```

Comment:
```suggestion
            if self.options.get_safe("use_sse4_2"):
```

Let's say we are using ARM arch here. The option use_sse4_2 will not be available and will break conan create command. Checking the arch should be done in `validate()` method, like:

```python
def validate(self):
    if self.options.get_safe("use_sse4_2") and str(self.settings.arch) not in ['x86', 'x86_64']:
        raise ConanInvalidConfiguration(f"{self.ref} can use the option use_sse4_2 only on x86 and x86_64 archs.")

Reply:
> 

Thanks.   fixed.